src/pages/_app.js

```diff
@@ -0,0 +1,16 @@
+import Layout from "~/components/Layout";
+import GlobalStyle from "../styles";
+import { events } from "~/lib/data";
+import { uniqueCities } from "~/utils/getCities";
+
+export default function App({ Component, pageProps }) {
+  console.log(eventsWithId);
+  return (
+    <>
+      <GlobalStyle />
+      <Layout>
+        <Component {...pageProps} events={events} cities={uniqueCities} />
```

Comment:
as long as your date is coming from a static file you could consider not passing it down via props but instead importing it everywhere you need. Doing it as you do right now might prepare you for turning it into state at a later point. Your call…

```suggestion
        <Component {...pageProps} events={events} cities={uniqueCities} />
```